### PR TITLE
🌐 Spanish translation of help-fastapi.md

### PR DESCRIPTION
**Pull request by pedabraham**
Following with the Spanish translations in #1218 , I made the translation of help-fastapi.md

This pull request includes:

help-fastapi.md translation
Update of docs/es/mkdocs.yml

@mariacamilagl , I will wait for revision

### CODE REVIEW

File: docs/es/docs/help-fastapi.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+* Corregir un error que encuentres en la documentación.
+* Compartir un artículo, video, o podcast que tu creaste o encontraste sobre FastAPI por medio de <a href="https://github.com/tiangolo/fastapi/edit/master/docs/en/data/external_links.yml" class="external-link" target="_blank">editar esté archivo</a>.
+    * Asegurate de añadir un enlace al final de la sección correspondiente.
+* Ayudar a [traducir esta documentación](contributing.md#translations){.internal-link target=_blank} a tu idioma.
+    * Puedes también ayudar a revisar las traducciones creadas por otros.
+* Proponer nuevas secciones en la documentación.
+* Ayudar a resolver un existente issue/bug.
+* Añadir una nueva característica.
+
+## Unete al chat
```
Comment (eavv) [inline docs/es/docs/help-fastapi.md:102]:
```suggestion
## Únete al chat
```

File: docs/es/docs/help-fastapi.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+Unete al 👥 <a href="https://discord.gg/VQjSZaeJmf" class="external-link" target="_blank">chat en Discord</a> 👥 y convive con otros dentro de la comunidad FastAPI.
```
Comment (eavv) [inline docs/es/docs/help-fastapi.md:104]:
```suggestion
Únete al 👥 <a href="https://discord.gg/VQjSZaeJmf" class="external-link" target="_blank">chat en Discord</a> 👥 y convive con otros dentro de la comunidad FastAPI.
```

File: docs/es/docs/help-fastapi.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+!!! Consejo
+    Para preguntas, hazlas en los <a href="https://github.com/tiangolo/fastapi/issues/new/choose" class="external-link" target="_blank">issues de GitHub</a>, hay una mayor posibilidad de recibir ayuda por [Expertos en FastAPI](fastapi-people.md#experts){.internal-link target=_blank}.
+
+    Trata de mantener el chat solo para otras conversaciones generales.
+
+También existe el anterior <a href="https://gitter.im/tiangolo/fastapi" class="external-link" target="_blank">chat en Gitter</a>, pero como no tiene canales o funciones avanzadas las conversaciones son mas complicadas, así que Discord es nuestra actual recomendación.
```
Comment (eavv) [inline docs/es/docs/help-fastapi.md:111]:
```suggestion
También existe el anterior <a href="https://gitter.im/tiangolo/fastapi" class="external-link" target="_blank">chat en Gitter</a>, pero como no tiene canales o funciones avanzadas las conversaciones son más complicadas, así que Discord es nuestra actual recomendación.
```

File: docs/es/docs/help-fastapi.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+### No uses el chat para preguntas
+
+Ten en cuenta que el chat permite "conversaciones mas libres", así que puede ser fácil que al hacer una pregunta bastante general y/o complicada no haya una respuesta.
```
Comment (eavv) [inline docs/es/docs/help-fastapi.md:115]:
```suggestion
Ten en cuenta que el chat permite "conversaciones más libres", así que puede ser fácil que al hacer una pregunta bastante general y/o complicada no haya una respuesta.
```

File: docs/es/docs/help-fastapi.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+En los issues de GitHub la plantilla puede ayudarte a redactar una pregunta correctamente, para que asi puedas obtener una buena respuesta de manera mas sencilla, o incluso resolver el problema por ti mismo antes de pedir ayuda. De esta manera puedo asegurarme que siempre pueda responder a todo, incluso si toma algo de tiempo. Personalmente no puedo hacer esto con los sistemas de chat.  😅
```
Comment (eavv) [inline docs/es/docs/help-fastapi.md:117]:
```suggestion
En los issues de GitHub la plantilla puede ayudarte a redactar una pregunta correctamente, para que así puedas obtener una buena respuesta de manera más sencilla, o incluso resolver el problema por ti mismo antes de pedir ayuda. De esta manera puedo asegurarme que siempre pueda responder a todo, incluso si toma algo de tiempo. Personalmente no puedo hacer esto con los sistemas de chat.  😅
```

File: docs/es/docs/help-fastapi.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+
+Las conversaciones en los sistemas de chat no permiten una búsqueda sencilla como GitHub, por lo que las preguntas y las respuestas pueden perderse en la conservación. Y solo las que están en los issues de GitHub cuentan para convertirse en un [FastAPI Expert](fastapi-people.md#experts){.internal-link target=_blank}, asi que es mas probable que recibas mas atención en los issues de GitHub.
+
+Por otro lado hay miles de usuarios en los sistemas de chat, así que existe una alta posibilidad de que exista alguien que te pueda ayudar ahi, casi todo el tiempo. 😄
```
Comment (eavv) [inline docs/es/docs/help-fastapi.md:121]:
```suggestion
Por otro lado, hay miles de usuarios en los sistemas de chat, así que existe una alta posibilidad de que exista alguien que te pueda ayudar ahi, casi todo el tiempo. 😄
```

File: docs/es/docs/help-fastapi.md [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+
+## Patrocina al autor
+
+También puedes ayudar financieramente al autor, mediante <a href="https://github.com/sponsors/tiangolo" class="external-link" target="_blank">GitHub sponsors</a>.
+
+Ahi puedes comprarme un cafe ☕️ para decir gracias. 😄
```
Comment (eavv) [inline docs/es/docs/help-fastapi.md:127]:
```suggestion
Ahí puedes comprarme un café ☕️ para decir gracias. 😄
```

File: docs/es/docs/help-fastapi.md [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+Como has visto en la documentacion, FastAPI está sobre hombros de gigantes, Starlette y Pydantic.
+
+También los puedes patrocinar:
+
+* <a href="https://github.com/sponsors/samuelcolvin" class="external-link" target="_blank">Samuel Colvin (Pydantic)</a>
+* <a href="https://github.com/sponsors/encode" class="external-link" target="_blank">Encode (Starlette, Uvicorn)</a>
+
+---
+
+Gracias! 🚀
```
Comment (eavv) [inline docs/es/docs/help-fastapi.md:142]:
```suggestion
¡Gracias! 🚀
```

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**